hooks/boot/10-reset_data.rb

```diff
@@ -1,8 +1,7 @@
-# Add reset option
-app_option(
-  '--reset',
+add_option(
```

Comment:
The tests are failing on this. It's, despite your intuition, `app_option`. But I totally understand thinking `add_option` should be the name.

Reply:
There's something to say for both. One is declarative, the other imperative. Given the rest of the hooks I think imperative feels more in line with the rest.